### ISSUE TITLE
False-positive ERA001 with `# nuitka-project:` configuration

### ISSUE THREAD

**Issue by bersbersbers** [charliermarsh/ruff#4618]
https://github.com/Nuitka/Nuitka allows in-file configurations:

```python
# nuitka-project: --onefile
# nuitka-project: --onefile-windows-splash-screen-image={MAIN_DIRECTORY}/Splash-Screen.png

# Whatever this is obviously
print("Delaying startup by 10s...")
import time, tempfile, os
time.sleep(10)

# Use this code to signal the splash screen removal.
if "NUITKA_ONEFILE_PARENT" in os.environ:
   splash_filename = os.path.join(
      tempfile.gettempdir(),
      "onefile_%d_splash_feedback.tmp" % int(os.environ["NUITKA_ONEFILE_PARENT"]),
   )

   if os.path.exists(splash_filename):
      os.unlink(splash_filename)

print("Done... splash should be gone.")
...

# Rest of your program goes here.
```

I find lines starting with `# nuitka-project: ` should not be flagged as ERA001 (and not be removed by `--fix`).

(If there was an option to disable ERA001 for certain keywords, users could configure that themselves.)

**Comment by bersbersbers** [comment]
Alternatively, being able to disable ERA001 for this block would also be an option. I know how to suppress it for a single line or the whole file, but not a block.

**Comment by JonathanPlasse** [comment]
You can use [task-tags](https://beta.ruff.rs/docs/settings/#task-tags)
```toml
task-tags = ["nuitka-project"]
```

**Comment by bersbersbers** [comment]
> You can use [task-tags](https://beta.ruff.rs/docs/settings/#task-tags)

This is great! It would be even greater if that was documented at https://beta.ruff.rs/docs/rules/commented-out-code/ and https://beta.ruff.rs/docs/rules/line-too-long/, then I would have found it without opening this issue :)

**Comment by JonathanPlasse** [comment]
Would you be open to making a pull-request for this?

**Comment by bersbersbers** [comment]
@JonathanPlasse sure, see #4644.

**Comment by charliermarsh** [comment]
Thank you!
